### PR TITLE
DDF-2132 Cleans up console commands

### PR DESCRIPTION
**Pull request by vinamartin**
#### What does this PR do?
-Reads from command line correctly
-Adds in missing dependency that was getting pulled in through another dependency that will need to be removed with the karaf upgrade
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@emmberk @clockard 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@clockard
@lessarderic

### CODE REVIEW

File: platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/MigrationCommands.java [hunk 1]
```diff
@@ -18,7 +18,6 @@
 import org.apache.karaf.shell.api.action.Action;
 import org.fusesource.jansi.Ansi;
 import org.fusesource.jansi.Ansi.Attribute;
-
```
Comment (clockard) [inline platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/MigrationCommands.java]: doesn't look like there is any real change for this file.

File: catalog/core/catalog-core-eventcommands/pom.xml [hunk 2]
```diff
@@ -53,6 +53,11 @@
             <groupId>joda-time</groupId>
             <artifactId>joda-time</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.fusesource.jansi</groupId>
+            <artifactId>jansi</artifactId>
```
Comment (vinamartin) [inline catalog/core/catalog-core-eventcommands/pom.xml]: This is what karaf uses

File: platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/MigrationCommands.java (hunk 1 above)
Comment (vinamartin) [inline platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/MigrationCommands.java]: I can remove it from the PR

**Comment by vinamartin** [comment]
build now

**Comment by vinamartin** [comment]
build now

**Comment by clockard** [comment]
build now

**Pull request merged**